gap_hci.go

```diff
@@ -341,10 +342,13 @@ func (a *Advertisement) Configure(options AdvertisementOptions) error {
 	}
 
 	a.serviceUUIDs = append([]UUID{}, options.ServiceUUIDs...)
-	a.interval = uint16(options.Interval)
-	if a.interval == 0 {
-		a.interval = 0x0800 // default interval is 1.28 seconds
+	if options.Interval == 0 {
+		// Pick an advertisement interval recommended by Apple (section 35.5
+		// Advertising Interval):
+		// https://developer.apple.com/accessories/Accessory-Design-Guidelines.pdf
+		options.Interval = NewDuration(152500 * time.Microsecond) // 152.5ms
```

Comment:
For some reason I feel bad about changing input data, would prefer how it was done previously but the value?

I do understand it's the same and value doe not leak back (not a pointer, not now), I just have natural aversion to changing input data :)

Reply:
I took this code verbatim from https://github.com/tinygo-org/bluetooth/blob/release/gap_nrf528xx-advertisement.go#L44-L49 where are are already using it. :)